docs/recipes/ExpoRouter.md

```diff
@@ -0,0 +1,1806 @@
+
+<details>
+  <summary>src/app/(app)/index.tsx</summary>
+
+```tsx
+import { router } from "expo-router";
+import { observer } from "mobx-react-lite";
+import React from "react";
+import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native";
+import { Button, Text } from "src/components";
+import { isRTL } from "src/i18n";
+import { useStores } from "src/models";
+import { colors, spacing } from "src/theme";
+import { useHeader } from "src/utils/useHeader";
+import { useSafeAreaInsetsStyle } from "src/utils/useSafeAreaInsetsStyle";
+
+const welcomeLogo = require("assets/images/logo.png");
+const welcomeFace = require("assets/images/welcome-face.png");
+
+export default observer(function WelcomeScreen() {
+  const {
+    authenticationStore: { logout },
+  } = useStores();
+
+  function goNext() {
+    router.replace("/showroom");
+  }
+
+  useHeader(
+    {
+      rightTx: "common.logOut",
+      onRightPress: logout,
+    },
+    [logout]
+  );
+
+  const $bottomContainerInsets = useSafeAreaInsetsStyle(["bottom"]);
+
+  return (
+    <View style={$container}>
+      <View style={$topContainer}>
+        <Image style={$welcomeLogo} source={welcomeLogo} resizeMode="contain" />
+        <Text
+          testID="welcome-heading"
+          style={$welcomeHeading}
+          tx="welcomeScreen.readyForLaunch"
+          preset="heading"
+        />
+        <Text tx="welcomeScreen.exciting" preset="subheading" />
+        <Image style={$welcomeFace} source={welcomeFace} resizeMode="contain" />
+      </View>
+
+      <View style={[$bottomContainer, $bottomContainerInsets]}>
+        <Text tx="welcomeScreen.postscript" size="md" />
+        <Button
+          testID="next-screen-button"
+          preset="reversed"
+          tx="welcomeScreen.letsGo"
+          onPress={goNext}
+        />
+      </View>
+    </View>
+  );
+});
+
+const $container: ViewStyle = {
+  flex: 1,
+  backgroundColor: colors.background,
+};
+
+const $topContainer: ViewStyle = {
+  flexShrink: 1,
+  flexGrow: 1,
+  flexBasis: "57%",
+  justifyContent: "center",
+  paddingHorizontal: spacing.lg,
+};
+
+const $bottomContainer: ViewStyle = {
+  flexShrink: 1,
+  flexGrow: 0,
+  flexBasis: "43%",
+  backgroundColor: colors.palette.neutral100,
+  borderTopLeftRadius: 16,
+  borderTopRightRadius: 16,
+  paddingHorizontal: spacing.lg,
+  justifyContent: "space-around",
+};
+const $welcomeLogo: ImageStyle = {
+  height: 88,
+  width: "100%",
+  marginBottom: spacing.xxl,
+};
+
+const $welcomeFace: ImageStyle = {
+  height: 169,
+  width: 269,
+  position: "absolute",
+  bottom: -47,
+  right: -80,
+  transform: [{ scaleX: isRTL ? -1 : 1 }],
+};
+
+const $welcomeHeading: TextStyle = {
+  marginBottom: spacing.md,
+};
+```
+
+</details>
+
```

Comment:
I feel like this is a point where we should validate that things are working. The app has several things that need to be updated before it will come to life here.

Once you do get the application running, clicking login will keep you on the login screen as the login function only sets the `authToken`. If you reload the application, it will take you to the welcome screen.

Reply:
👍 
addressed with this "Checkpoint" section, and I fixed the auth issue in the log-in code by imperatively navigating to login after clicking the button